web/src/features/fbaCalculator/RowManager.ts

```diff
@@ -233,11 +233,13 @@ export class RowManager {
         formattedWindSpeed = value.wind_speed.toFixed(DECIMAL_PLACES)
       }
       rowString.push(formattedWindSpeed)
-      rowString.push(
-        isUndefined(value.precipitation) || isNull(value.precipitation)
-          ? ''
-          : value.precipitation.toFixed(DECIMAL_PLACES)
-      )
+      let formattedPrecip = ''
+      if (!isUndefined(value.precip) && !isNaN(value.precip) && !isNull(value.precip)) {
```

Comment:
Can we use `isNil` here?

Reply:
Done in https://github.com/bcgov/wps/pull/4433/commits/9139d9ee9b06aa44bc44f3949a1aa95ecd7d1a2d